### PR TITLE
Adds tracking for support page CTAs

### PR DESCRIPTION
**Pull request by lucymonie**
## Why are you doing this?
The aim is to improve visibility of clicks on support page call-to-action buttons.

[**Trello Card**](https://trello.com/c/AZktsdvl/2454-dev-add-ga-tracking-to-subscribe-and-contribute-cta-buttons-in-why-support-page)

### CODE REVIEW

File: support-frontend/assets/pages/showcase/components/ctaSubscribe.jsx [hunk 1]
```diff
@@ -22,7 +23,14 @@ export default function CtaSubscribe() {
         </p>
       </Text>
       <NarrowContent>
-        <AnchorButton aria-label={null} icon={<ArrowRightStraight />} href="/subscribe">Choose a Subscription</AnchorButton>
+        <AnchorButton
+          aria-label={null}
+          icon={<ArrowRightStraight />}
+          href="/subscribe"
+          onClick={() => sendClickedEvent('support_page_cta_subscribe')()}
```
Comment (rupertbates) [inline support-frontend/assets/pages/showcase/components/ctaSubscribe.jsx]: There is also a function called `sendTrackingEventsOnClick` in subscriptions.js which does a very similar thing but also sends data to Ophan. Not sure which is the best bet in this case, probably the one you've picked?

Comment (lucymonie) [reply support-frontend/assets/pages/showcase/components/ctaSubscribe.jsx]: Yes - I saw that function when I was reviewing your PR! I think it's less suitable as it wants you to specify a product as opposed to clicking on 'subscribe' or 'contribute'

File: support-frontend/assets/pages/showcase/components/ctaContribute.jsx [hunk 2]
```diff
@@ -42,6 +43,7 @@ export default function CtaContribute() {
               icon={<ArrowRightStraight />}
               appearance="secondary"
               href="/contribute"
+              onClick={() => sendClickedEvent('support_page_cta_contribute')()}
```
Comment (rupertbates) [inline support-frontend/assets/pages/showcase/components/ctaContribute.jsx]: Are you going to put a similar event on the subscribe button?

Comment (lucymonie) [reply support-frontend/assets/pages/showcase/components/ctaContribute.jsx]: Yes, I have done :-)

Comment (lucymonie) [reply support-frontend/assets/pages/showcase/components/ctaContribute.jsx]: See below :down

Comment (rupertbates) [reply support-frontend/assets/pages/showcase/components/ctaContribute.jsx]: 🤦‍♂ Oh yeah!

**Pull request merged**